conf/cron

```diff
@@ -1 +1 @@
-10 * * * * root curl --request POST --url 'http://127.0.0.1:__PORT__/api/cronjob/checkStatus' --header 'Authorization: Bearer __CRON_KEY__' ; curl --request POST --url 'http://127.0.0.1:__PORT__/api/cronjob/getStorageUsed' --header 'Authorization: Bearer __CRON_KEY__'
+10 * * * * root curl --request POST --url 'http://127.0.0.1:__PORT__/api/v1/cron/status' --header 'Authorization: Bearer __CRON_KEY__' ; curl --request POST --url 'http://127.0.0.1:__PORT__/api/v1/cron/storage' --header 'Authorization: Bearer __CRON_KEY__'

```

Comment:
Maybe we want to replace running this as the dedicated user instead of `root`.

Otherwise this looks to me like a solution.

@ericgaspar , I don’t have the permission (nor maybe should) to merge this, but if you merge this (to `testing` if needed), I’m happy to test it.